libse/Subtitle.cs

```diff
@@ -580,5 +580,21 @@ public string GetFastHashCode()
             return sb.ToString().TrimEnd();
         }
 
+        public string GetAllText(bool appendNewLine)
+        {
+            var sb = new StringBuilder();
+            foreach (Paragraph p in Paragraphs)
+            {
+                if (appendNewLine)
+                {
+                    sb.AppendLine(p.Text);
+                }
+                else
+                {
+                    sb.Append(p.Text.TrimEnd() + ' ');

```

Comment:
Oops, obviously, I wasn't thinking clearly. `Paragraph.Text` may contain line breaks. Hence, the return statement should probably be something like:
```
sb.Replace("\r\n", " ").Replace('\r', ' ').Replace('\n', ' ').ToString().TrimEnd()
```